python/ray/tune/function_runner.py

```diff
@@ -516,11 +516,15 @@ def _report_thread_runner_error(self, block=False):
             pass
 
 
-def wrap_function(train_func, warn=True):
+def wrap_function(train_func, durable=False, warn=True):
+    inherit_from = (FunctionRunner, )
+
     if hasattr(train_func, "__mixins__"):
-        inherit_from = train_func.__mixins__ + (FunctionRunner, )
-    else:
-        inherit_from = (FunctionRunner, )
+        inherit_from = train_func.__mixins__ + inherit_from
```

Comment:
does this mean durable() is not compatible with `with_parameters`?

Reply:
`with_parameters` returns a function, so calling `tune.durable(tune.with_parameters(fn))` should work. I'll test this, and try to detect it to throw a meaningful error message if this is mixed up.

Reply:
Confirmed it works